### PR TITLE
Copy to Clipboard

### PR DESCRIPTION
**Pull request by nikolaswise**
Stub some best practices, get copy to clipboard in there, start working the top nav a little bit with input from developers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,11 +55,6 @@
       "docs/"
     ]
   },
-  "babel": {
```
Comment (nikolaswise) [inline package.json]: babel wont compile unless this is gone. But it wont compile for you @paulcpederson  unless this is here?

File: docs/source/guides/best-practices/forms.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Best Practices
+description: Guidelines for great design decisions.
+layout: layouts/_designguide:text
+---
+
```
Comment (paulcpederson) [inline docs/source/guides/best-practices/forms.md]: You have a bug fix for the docs site in this pr, but I'm assuming you don't want this to go out yet, right? So should this best-practices stuff be in a branch? Or do you think we should just have it be a todo thing and deploy anyways?

File: package.json (hunk 1 above)
Comment (paulcpederson) [inline package.json]: I just deleted it locally and it seems like it still rolls up fine, so I guess delete it?

Comment (paulcpederson) [inline package.json]: Actually, nevermind, I just checked why I added that from git blame and it was when I added the `dist` task. Can you run `npm run dist` without that? It was minification that exploded not rollup.

Did you nuke rollup and install it again? Maybe you have an old version?

Comment (paulcpederson) [inline package.json]: Also, what does babel say when it explodes?

Comment (nikolaswise) [inline package.json]: Looks like removing this for sure breaks `npm run dist` on minification. Adding it back in for me creates the error: 
`It looks like your Babel configuration specifies a module transformer. Please disable it.`

Can you run all things happily with this in JSON?

File: docs/source/guides/best-practices/forms.md (hunk 2 above)
Comment (nikolaswise) [inline docs/source/guides/best-practices/forms.md]: Its orphan content, not linked to so not discoverable when pushed. This at least gets it into a space where we can both work on it.

Comment (paulcpederson) [inline docs/source/guides/best-practices/forms.md]: word up

File: package.json (hunk 1 above)
Comment (paulcpederson) [inline package.json]: yes. I can run `rollup` and `dist` with this in `package.json`

Comment (nikolaswise) [inline package.json]: Okay, I will put this back and figure out my tooling at my end.

Comment (paulcpederson) [inline package.json]: try adding `babelrc: false` maybe https://github.com/rollup/rollup-plugin-babel/issues/35 ?

**Pull request merged**